src/SYN.py

```diff
@@ -4381,6 +4373,10 @@ def LOGIC_IS_ZERO_DELAY(logic, parser_state, allow_none_delay=False):
         return True
     elif logic.func_name.startswith(C_TO_LOGIC.PRINTF_FUNC_NAME):
         return True
+    elif (SYN_TOOL == GOWIN) and logic.func_name.startswith("UNARY_OP_NOT_"):
```

Comment:
Also minor thing: If you dont mind, can you use the function name string consts from
https://github.com/JulianKemmerer/PipelineC/blob/bd55aa790e761ddb15051247dcab172189d02ccd/src/C_TO_LOGIC.py#L44
the prefix and not op name combined with trailing underscore